### PR TITLE
Added test to check duplicate rebuses

### PR DESCRIPTION
**Pull request by devmaximilian**
Associated Issue: #144 

### Summary of Changes

- Add a test case that makes sure that there are no duplicated rebuses
- Removed duplicate rebuses

### CODE REVIEW

File: tests/rebus.spec.js [hunk 1]
```diff
@@ -58,4 +58,23 @@ describe('Tests for rebuses', () => {
...
+            if (
+              rebus.words.toString().toLowerCase() ===
+              potentialDuplicate.words.toString().toLowerCase()
+            ) {
+              duplicates.push(potentialDuplicate.words);
+            }
+          }
+        });
+      });
+      expect(duplicates).toEqual(new Array(0));
```
Comment (ollelauribostrom) [inline tests/rebus.spec.js]: Just do `.toEqual([])` 🙂

File: tests/rebus.spec.js [hunk 2]
```diff
@@ -58,4 +58,23 @@ describe('Tests for rebuses', () => {
...
+      const rebuses = getRebuses();
+      const duplicates = [];
+      rebuses.forEach(rebus => {
+        rebuses.forEach(potentialDuplicate => {
+          if (rebus !== potentialDuplicate && !duplicates.includes(rebus.words)) {
+            if (
+              rebus.words.toString().toLowerCase() ===
+              potentialDuplicate.words.toString().toLowerCase()
+            ) {
+              duplicates.push(potentialDuplicate.words);
```
Comment (ollelauribostrom) [inline tests/rebus.spec.js]: Maybe push the whole rebus object to to `duplicates` array to make it even easier to debug failing tests (and then check for `!duplicates.includes(rebus)`)

Comment (devmaximilian) [reply tests/rebus.spec.js]: Done! 😄

**Comment by ollelauribostrom** [comment]
Awesome, thank you!

**Pull request merged**